src/components/atoms/AdminRestricted/AdminRestricted.scss

```diff
@@ -12,7 +12,7 @@ $denied-shape-size: 96px;
     margin: 0;
     padding: 0;
     // in case there are other considerations, calc can help, example:
-    //width: calc(100% - #{$spacing--xs});
+    width: calc(100% - #{$spacing--xs});
```

Comment:
Allowing it for now, hopefully deleted soon with PR https://github.com/sparkletown/sparkle/pull/2345